Content.Server/Radio/EntitySystems/RadioSystem.cs

```diff
@@ -273,11 +294,13 @@ private string WrapRadioMessage(
             boldId = language.SpeechOverride.FontId;
         // Goobstation Edit - end
 
+        var fontSize = fontSizeOverride ?? language.SpeechOverride.FontSize ?? speech.FontSize; // Goobstation loud command
+
```

Comment:
If you're set on using your code, it could look something like this.

```suggestion
        // Goobstation start - Loud command
        int? fontSizeIncrease = null;
        if (TryComp<EncryptionKeyHolderComponent>(radioSource, out var holder))
            foreach (var ent in holder.KeyContainer.ContainedEntities)
                if (TryComp<EncryptionKeyComponent>(ent, out var key)
                    && key.Channels.Contains(channel.ID))
                {
                    fontSizeIncrease = key.FontSizeIncrease;
                    break;
                }
                
        var fontSize = speech.FontSize;
        if (fontSizeIncrease == null)
            fontSize = language.SpeechOverride.FontSize ?? fontSize;
        // Goobstation endcommand

```

Reply:
This isn't great and I personally would do something different, but I'll mention it in the overhead comment instead of this one.